fiftyone/factory/repos/delegated_operation_doc.py

```diff
@@ -127,14 +131,24 @@ def from_pymongo(self, doc: dict):
             if "updated_at" in doc["status"]:
                 self.status.updated_at = doc["status"]["updated_at"]
 
+        if "pipeline" in doc and doc["pipeline"] is not None:
```

Comment:
since you just raise an error later, should you even be setting it?

Reply:
Attempt to minimize oss-teams delta